packages/vue-instantsearch/src/components/CurrentRefinements.vue

```diff
@@ -17,7 +17,7 @@
             :item="item"
             :createURL="state.createURL"
           >
-            <span :class="suit('label')">{{ capitalize(item.label) }}: </span>
+            <span :class="suit('label')">{{ capitalize(item.label) }}:</span>
```

Comment:
will the refinement now not look like `attribute:value` instead of `attribute: value`?

Reply:
Yes, this mirrors what we have for JS / React, and (in our demos) the spacing is set with the default CSS rules from algolia/satellite.

Reply:
I think I'd prefer js & react to conform to Vue here, allowing for better copy/paste too for example

Reply:
You're right. I've updated this in https://github.com/algolia/instantsearch/pull/5838/commits/4941b50c2a9d34a70a238a50aa2e9b9cd1b4788d.